### PR TITLE
Revert "Revert "Remove the dependency on `chrono`""

### PR DESCRIPTION
**Pull request by julesdesmit**
Reverts AleoHQ/aleo-setup#451

### CODE REVIEW

File: phase1-coordinator/src/environment.rs [hunk 1]
```diff
@@ -225,21 +224,17 @@ pub struct Environment {
     /// Returns the maximum duration a contributor can go without
     /// being seen by the coordinator before it will be dropped from
     /// the ceremony by the coordinator.
-    #[serde_as(as = "DurationSecondsWithFrac<String>")]
-    contributor_seen_timeout: chrono::Duration,
+    contributor_seen_timeout: time::Duration,
```
Comment (niklaslong) [inline phase1-coordinator/src/environment.rs:227]: Note: these won't be serialised in the same way when/if they are written to a file. It used to be something close to rfc3339 and now it'll be a unix timestamp. Seems like this isn't a concern but certainly worth pointing out.

File: phase1-coordinator/src/testing/resources/round.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "version": 0,
     "height": 0,
-    "startedAt": "1970-01-01T00:01:01Z",
+    "startedAt": 61,
```
Comment (niklaslong) [inline phase1-coordinator/src/testing/resources/round.json:4]: Case in point ☝️ 
@julesdesmit this sounds familiar to what you were saying the issue was?

File: phase1-coordinator/src/environment.rs (hunk 1 above)
Comment (julesdesmit) [reply phase1-coordinator/src/environment.rs:227]: Yeah I already updated the coordinator json files to account for this. It broke some stuff but the fix was trivial :)

File: phase1-coordinator/src/testing/resources/round.json (hunk 2 above)
Comment (julesdesmit) [reply phase1-coordinator/src/testing/resources/round.json:4]: Yep, that caused the disconnect with the state monitor! I guess we are using a different type somewhere.

**Review by niklaslong — APPROVED** [review]
LGTM so long as that type issue won't break things again @julesdesmit?

**Comment by julesdesmit** [comment]
@niklaslong shouldn't be an issue, the current coordinators were deployed with the new `time` dependency and everything should be compatible going forwards :)

**Pull request merged**